client/.eslintrc.js

```diff
@@ -23,6 +23,11 @@ module.exports = {
   env: {
     browser: true,
     es6: true,
+    commonjs: true
```

Comment:
No, not too many, only one :)

In ESLint docs, it says this about the `commonjs` environment:

> commonjs - CommonJS global variables and CommonJS scoping (use this for browser-only code that uses Browserify/WebPack).

So since we are actually using WebPack, is there any reason why we shouldn't use this environment?